src/main/java/net/mcreator/workspace/references/ReferencesFinder.java

```diff
@@ -80,7 +80,7 @@ public static Set<ModElement> searchModElementUsages(Workspace workspace, ModEle
 					procedureXmlPattern.matcher(t).find()

```

Comment:
PR would be shorter and cleaner if you did this in getGeneratableElements helper method

Reply:
✔️ Fixed